JavaScript/Github_Fetch_Issue_Details/issueDetails.js

```diff
@@ -0,0 +1,67 @@
+const express = require("express");
+const axios = require("axios");
+const fs = require("fs");
+const getDetails = async (username, repoName) => {
+  const url = `https://api.github.com/repos/${username}/${repoName}/issues`;
+  await axios
+    .get(url)
+    .then(async (result) => {
+      console.log(result);
+      result.data.map(async (issue) => {
+        const issueCreator = issue.user.login;
+        const issueAssignees = issue.assignees.map((assignee) => assignee);
+        const issueOpenDate = new Date(issue.created_at);
+        const issueState = issue.state;
+        const issueClosedDate =
+          issueState === "open"
+            ? "Its not closed yet"
+            : new Date(issue.closed_at);
+        const issueContent = issue.body;
+        let issueCommentorDetails;
+        // debugger
+        const commentsUrl = url + "/" + issue.number + "/comments";
```

Comment:
It will be better to use `backticks` here as well

Reply:
Sure, Will change it to backticks.